setup.py

```diff
@@ -14,6 +14,16 @@
     os.system('python setup.py sdist upload')
     sys.exit()
 
+
+def required(sfx=''):
+    """ Load the requirements from the requirements.txt file"""
+    reqs = []
+    try:
+        with open(f"requirements{sfx}.txt") as f:
+            reqs = [ln.strip() for ln in f.readlines() if not ln.startswith('-') and not ln.startswith('#') and ln.strip() != '']
+    finally:
+        return reqs
```

Comment:
What's the point of wrapping this in a try if the exception isn't caught? This should fail when the file is missing.
```suggestion
    with open(f"requirements{sfx}.txt") as f:
        reqs = [ln.strip() for ln in f.readlines() if not ln.startswith('-') and not ln.startswith('#') and ln.strip() != '']

```

Reply:
I'll just add this suggestion